doc/source/whatsnew/v1.1.0.rst

```diff
@@ -336,68 +340,6 @@ Other enhancements
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-``MultiIndex.get_indexer`` interprets `method` argument differently
```

Comment:
where did this go?

Reply:
https://github.com/pandas-dev/pandas/pull/34865/files#diff-e763086372257ed7ca944d8bc5042a04R813

Reply:
I would rather not move this, this example is very needed to understand what the heck that change was.

I have no problem with you moving things around, but let's leave the examples. If you want to remove them, then let's discuss individually.